package.json

```diff
@@ -26,17 +26,21 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
+    "@types/hast": "^2.3.0",
```

Comment:
We've been putting `@types` dependencies as `devDependencies`
see https://github.com/unifiedjs/unified/issues/45 for some context

Reply:
I believe the general consensus in the NodeJS community is that `@types` dependencies should be installed is regular `dependencies`, if they are needed to consume the package in TypeScript. Since this package only exposes type definitions, not internal type definitions, the `@types` are definitely used and belong in `dependencies`.

That having said, I’m already glad I can use this package in TypeScript now.

Reply:
Agreed,

The challenge unified faces is two fold:
1. Unified packages are lower level and get included in higher level packages, often in a non-API driven way, through CLI tools like `remark-cli`, `rehype-cli`, and `prettier`.
2. Not all the projects are running in a Node environment, especially ones using CLis, many are browser based, and in a browser/webpack context sometimes the global types from `@types/node` can conflict with the the types used by a  project.

Currently Unified is working around this, by keeping `@types` as devDependencies to make sure no unwanted `@typed` get transitively included.
If you have suggestions on other approaches, feel free to chime in on unifiedjs/unified#45